main.go

```diff
@@ -60,6 +62,16 @@ func main() {
 
 	var rolebindingsResyncPeriod time.Duration
 
+	authTypes := []request.AuthType{
+		request.TLSCertificate,
+		request.BearerToken,
+	}
+
+	authTypesMap := map[request.AuthType][]string{
+		request.BearerToken:    {request.BearerToken.String(), "bearer", "token"},
+		request.TLSCertificate: {request.TLSCertificate.String(), "certificate", "cert"},
```

Comment:
Do we really need multiple values for these enums? It could create confusion and I would stick to the `String` representation.

Reply:
ok, i've reduce these lists to just the string repr for each